R/stroke.R

```diff
@@ -191,28 +193,42 @@ get_best_link <- function(angles, links, angle_threshold = 0) {
 }
 
 #' @noRd
-cross_check_links <- function(best_links, flow_mode = FALSE) {
+check_reciprocal <- function(links, best_links, colname) {
```

Comment:
Maybe `side` instead of `colname` (this can be either "start" or "end")? Also I think we should remove `links` from here, see comment below:
```suggestion
check_reciprocal <- function(best_links, side) {
```

Reply:
Ah, I was looking for a good word and could not find one :) `side` is good!